app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -574,6 +574,7 @@ class SmokeTest {
 
     @Test
     // Saves a login, then changes it and verifies the update
+    @Ignore("Failing on https://github.com/mozilla-mobile/fenix/pull/20573")

```

Comment:
nit: Would be better to link to the ticket which in turn can reference where it failed.